lib/resources/order-recipients.js

```diff
@@ -16,6 +16,7 @@ const { log, prettyPrint } = require('../utils/logger');
 
 const ordersClient = axios.create();
 
+// TODO: add request object to callback when errors occur (for API module as well).
```

Comment:
Can we add a ticket to this effect for the overhaul Epic?

Reply:
Done, see https://rewardops.atlassian.net/browse/MX-1559